### ISSUE TITLE
Tweakscale support? 

### ISSUE THREAD

**Issue by shagg-e** [jbengtson/RealISRU#33]
As of right now on the 1.6.1 builds of RO/RP1, to produce any usable amount of fuel in a meaningful amount of time, a large number of parts are necessary. Whether with Tweakscale itself, or through preset scaled sizes like Infernal Robotics Next does it these days, it would be a big QoL feature for users and in-game performance to, say, only need 1 or 2 electrolysis units rather than spam huge arrays of them, connect different landed modules using KIS/KAS and massively increasing part count and resultant game lag because of it. Or, conversely, having a toggleable option to dump more power into a unit to increase its productivity (up to a certain point, providing you can radiate the heat away), but I'm not sure if that's based in any level of realism with the actual IRL processes this mod is based from.

**Comment by Starwaster** [comment]
> Or, conversely, having a toggleable option to dump more power into a unit to increase its productivity (up to a certain point, providing you can radiate the heat away), but I'm not sure if that's based in any level of realism with the actual IRL processes this mod is based from.

Yes and no. With regards to power, these processes are fairly linear. But you can't just take a given processor/reactor and double its power input. To convert more quantity it's also going to be a larger more massive unit. The mass increase wouldn't be linear;  a large scale unit would mass less per  given volume of resource being processed. Not sure how that would be written for TweakScale; those configs should be done by someone more knowledgeable.

In addition to the scale problem (and maybe compounded by it) is the issue of having to have multiple resource converters to get net product. I.e. for CH4+LOX, you need two harvesters (with builtin converters to produce water) + electrolysis + sabatier + liquefier. Between each 'layer' there are losses, especially with low output converters, and those losses scale over time.

The mod  has configs for combo converters which combine the electrolysis + sabatier + liquefier and they provide four parts of varying output. The performance of the combo units is greater than the equivalent separate converters combined because you don't have all those frame losses when doing water -> O2 + H2 -> sabatier + liquefier / O2 + liquefier; it's just all one unit that takes water + CO2 -> CH4 + O2 + water

BUT... there's no version of those converters that work with stock. Only TAC. When I get a chance I'll write up versions that work with the stock converters.
